examples/runqslower/src/bpf/runqslower.bpf.c

```diff
@@ -62,6 +66,16 @@ int handle__sched_wakeup_new(u64 *ctx)
 	return trace_enqueue(p->tgid, p->pid);
 }
 
+static inline long get_task_state(struct task_struct *ts)
+{
+	if (LINUX_KERNEL_VERSION < KERNEL_VERSION(5, 14, 0)) {
```

Comment:
see https://github.com/iovisor/bcc/pull/3659/files, there are better ways than using LINUX_KERNEL_VERSION check. Due to backporting, kernel version isn't always representative. CO-RE provides bpf_core_field_exists() check, we should use that here.